code/obj/zoldorf.dm

```diff
@@ -71,6 +23,9 @@ var/global/list/datum/zoldorfitem/zoldorf_items = list()
 	var/omen
 	var/occupied
 
+	var/list/datum/zoldorfitem/soul/soul_items = list()
+	var/list/datum/zoldorfitem/credit/credit_items = list()
```

Comment:
```
	var/list/datum/zoldorfitem/soul/soul_items = null
	var/list/datum/zoldorfitem/credit/credit_items = null
```
Then instantiate the lists in `New()`. Minor and especially so given the fact that this isn't made frequently, but code standard are code standards. I'm unclear on whether `/obj/machinery/playerzoldorf` could reasonably exist at startup, so this may be a non-issue, but moving them to `New` isn't going to be a problem in either case.

https://hackmd.io/@goonstation/docs/%2F%40goonstation%2Fcode#StartupRuntime-trade-offs-with-lists-and-the-hidden-init-proc